### DISCUSSION TITLE
pdk command to generate REFERENCE.md?

### DISCUSSION THREAD

**Discussion by jcpunk** [puppetlabs/pdk#1199] [Feature Request] [answered]
It would be handy if there was a quick way to generate the REFERENCE.md via PDK (probably under build?).  This would help folks keep it up to date with a trivial integration point.

**Comment by fr3nch13** [comment, marked as answer]
I know this is old, but the best I could find without installing anything other than PDK is this:

```
pdk bundle exec puppet strings generate --format markdown --out REFERENCE.md
```
